spec/plugin.spec.js

```diff
@@ -286,6 +286,45 @@ describe('CspHtmlWebpackPlugin', () => {
     );
   });
 
+  it('adds meta tag when no template is specified', done => {
```

Comment:
Maybe `adds meta tag with completed policy when the entire meta tag is missing`?